watcher.go

```diff
@@ -128,7 +134,9 @@ func (w *Watcher) Listen(listener Listener, roots ...string) {
 		}
 
 		// Else, walk the directory tree.
-		filepath.Walk(p, watcherWalker)
+		if err = filepath.Walk(p, watcherWalker); err != nil {
+			panic(err)
```

Comment:
Line 138 is part of Listen() that doesn't return anything, unlike watchWalker on line 89